heartbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,261 @@
+  mode: any <3>
+  timeout: 16s
+heartbeat.scheduler:
+  location: 'UTC-08:00' <4>
+output.elasticsearch:
+  hosts: ["myhost:9200"]
+----------------------------------------------------------------------
+<1> The `icmp` monitor is scheduled to run exactly every 5 seconds (10:00:00,
+10:00:05, and so on).
+<2> The `tcp` monitor is set to run every 5 seconds from the time when Heartbeat
+was started.
+<3> The `mode` specifies whether to ping one IP (`any`) or all resolvable IPs
+(`all`).
+<4> The `location` specifies the time zone for the scheduler.
++
+See <<heartbeat-configuration-details>> for a full description of each
+configuration option.
+
+. If you are sending output to Elasticsearch, set the IP address and port where
+Heartbeat can find the Elasticsearch installation:
++
+[source,yaml]
+----------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["192.168.1.42:9200"]
+----------------------------------------------------------------------
++
+If you are sending output to Logstash, see <<config-heartbeat-logstash>>
+instead.
+
+TIP: To test your configuration file, change to the directory where the
+Heartbeat binary is installed, and run Heartbeat in the foreground with
+the following options specified: +./heartbeat -configtest -e+. Make sure
+your config files are in the path expected by Heartbeat
+(see <<directory-layout>>). If you installed from DEB or RPM packages, run
++./heartbeat.sh -configtest -e+.
+
+[[heartbeat-template]]
+=== Step 3: Loading the Index Template in Elasticsearch
+
+:allplatforms:
+include::../../libbeat/docs/shared-template-load.asciidoc[]
+
+[[heartbeat-starting]]
+=== Step 4: Starting Heartbeat
+
+Start Heartbeat by issuing the appropriate command for your platform.
+
+*deb:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo /etc/init.d/ start
+----------------------------------------------------------------------
+
+*rpm:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo /etc/init.d/heartbeat start
+----------------------------------------------------------------------
+
+*mac:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo ./heartbeat -e -c heartbeat.yml -d "publish"
+----------------------------------------------------------------------
+
+*win:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+PS C:\Program Files\Heartbeat> Start-Service heartbeat
+----------------------------------------------------------------------
+
+By default, Windows log files are stored in +C:\ProgramData\heartbeat\Logs+.
+
+Heartbeat is now ready to send log files to your defined output.
+
+[[heartbeat-sample-dashboards]]
+=== Step 5: Loading Sample Kibana Dashboards
+
+//Steffen: Will we have Heartbeat dashboards for 5.2?
```

Comment:
Ok, I'll remove the section about dashboards for now and add a note to remind us to add it back in later.